packages/elements/src/pill/__test__/pill.test.js

```diff
@@ -282,4 +282,36 @@ describe('pill/Pill', () => {
       expect(el.pressed).to.equal(false, 'pressed property should stay false');
     });
   });
+
+  describe('Accessibility', () => {
+    it('should apply aria-pressed when toggle pill is pressed', async () => {
+      const el = await fixture(html`<ef-pill toggles></ef-pill>`);
+
+      setTimeout(() => el.dispatchEvent(new Event('tap')));
+      await oneEvent(el, 'tap');
+      expect(el.getAttribute('aria-pressed')).to.equal('true');
+
+      setTimeout(() => el.dispatchEvent(new Event('tap')));
+      await oneEvent(el, 'tap');
+      expect(el.getAttribute('aria-pressed')).to.equal('false');
+    });
+    it('aria-pressed should be removed when toggles attribute is removed', async () => {
```

Comment:
Should have line space between test cases for readability.

Reply:
I have already updated